lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -551,7 +551,7 @@ public function semanticalError($message = '', $token = null)
      * @param bool $resetPeek Reset peek after finding the closing parenthesis.
      *
      * @return mixed[]
-     * @psalm-return array{value: string, type: int|null|string, position: int}|null
+     * @psalm-return Token|null
```

Comment:
If anything is forcing to install Lexer 1.2.0 this won't work. `Token` wasn't in `AbstractLexer` yet.

Reply:
Forget about it. 1.2.3 is set in Composer.